packages/components/atoms/f-button/test-utils/component-objects/f-button.component.js

```diff
@@ -1,6 +1,6 @@
 const Page = require('@justeat/f-wdio-utils/src/page.object');
 
-module.exports = class Buttons extends Page {
+module.exports = class Button extends Page {
```

Comment:
Should this be renamed to be more specific to the type of button?

Reply:
could rename it to `ActionButton` potentially, but in storybook it's just called `Button` and the others are `Icon Button` and `Link Button` so it could probably stay as for consistency 😄

Reply:
could change to `ActionButton` potentially, but it's referenced as `Button` in storybook and the others are `Icon Button` and `Link Button` so we could also probably leave it for consistency